website/.env

```diff
@@ -0,0 +1 @@
+BEEHIIV_API_KEY=L4cvXwCqHADQ9DI4XQCHh7JSZIOE79qU7kBXysTOWmAnBWIXV3wzRfyTA12wt6Nn
```

Comment:
remove this one?

Reply:
Yeah, actually I've removed it from the entire git history, and now I'm force pushing it with over 200mb upload with my 5mb upload internet speed, so I deleted this, but still pushing it